src/scripting/ScriptStore.h

```diff
@@ -23,6 +24,7 @@ struct ScriptStore
     void TriggerOnOverlayOpen() const;
     void TriggerOnOverlayClose() const;
 
+    [[nodiscard]] TiltedPhoques::Vector<std::string> GetModNames() const;
```

Comment:
I'd probably make this into something like GetMods rather, which would return some mod info (would make it some table, for now, I suppose only name would be inside of it) and object normally returned by GetMod. Think it would be better personally, and we can extend it with proper mod info when we get to it.